package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@digicatapult/morello-ui",
-  "version": "0.1.1",
+  "version": "0.1.2",
   "description": "User interface for interacting with morello-api",
   "main": "src/index.js",
   "scripts": {
```

Comment:
Follow the convention for scripts as in other repos.

Reply:
Just copied most of them. I did not use react-script as they do have some vulnarabilities and install loads of stuff we won't even need so just did simple webpack config with loaders and etc. It has been updated